pages/index.tsx

```diff
@@ -95,26 +117,26 @@ export default function Home({ featuredStorefronts }: HomeProps) {
               </Button>
             </Space>
           </Marketing>
-          {heroStorefront && (
-            <Col xs={0} md={8}>
-              <StorePreview {...heroStorefront} />
-            </Col>
-          )}
-        </Section>
-        <FeaturedStores justify="center">
-          <Col>
-            <Title level={3}>Featured Creators</Title>
-            <List
-              grid={{ xs: 1, sm: 2, md: 3, lg: 3, xl: 3, xxl: 3, gutter: 16 }}
-              dataSource={drop<StorefrontFeature>(1, featuredStorefronts)}
-              renderItem={(feature) => (
-                <List.Item key={feature.storefront.subdomain}>
-                  <StorePreview {...feature} />
-                </List.Item>
-              )}
-            />
+          <Col xs={0} md={8}>
+            <FeaturedListingCarousel featuredListings={featuredListings} />
```

Comment:
For future work (and open for discussion) I don't like pulling things into seperate components until they are used in multiple spots. My personal ideal is just having the page component built out with component library.

Reply:
Yeah in general I agree. I created the component initially because I thought it would get more messy than it did 😅 Still for this one I recommend we keep the component as this section will probably be refactored in the future.